.travis/deb64

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
I think `bash` can be assumed to be in `/usr/bin` (or maybe `/bin`) by convention, no?